views/pages/ViewChartPage/index.jsx

```diff
@@ -116,6 +135,13 @@ const ViewChartPage = () => {
       setGraph(newGraph)
     })
   }, [chart_id, search])
+  const visibleRows = React.useMemo(
```

Comment:
why are we memoizing? I don't think this is helpful since we have complex objects in the dependency array.

Reply:
The dependencies we are tracking for this feature are required for pagination to work and display properly. If a page is changed and then a sort is used we need to trigger this logic to update what we display. I'm confused about what object looks complex

Reply:
it's not that they look complex, it is that they are "complex objects" and javascript equality checks will always treat them as unequal which makes this memoization useless and actually less performant than not having it at all.

Here is a contrived example:

![Screenshot 2024-03-20 at 2 37 18 PM](https://github.com/AMP-SCZ/dpdash/assets/2905145/74a665e4-a6ae-4f86-960d-c37e573810b5)

Reply:
oh okay, so with this commit we don't have to track the "complex objects" any more and we don't use safe navigators. 
[c014801](https://github.com/AMP-SCZ/dpdash/pull/695/commits/c0148010f3c405c14fa944ad28c4484d85267e02)